packages/kbn-types/README.md

```diff
@@ -0,0 +1,53 @@
+# External types for Kibana
+
+```
+npm install
+npm run build
+```
+
+This builds declaration files into `./types/packages/kbn-types`, which is
+specified as the "main" type file for this package in `./package.json`.
+
+Now external code can use this package, e.g. something like this:
+
+```js
```

Comment:
__optional nit:__ maybe `typescript` instead of `js`? Webstorm supports it (and also complains if we try to highlight typescript with `js` hint), but not sure if it's standard and your IDE supports it as well, so up to you.